resources/RULES.md

```diff
@@ -23,4 +23,4 @@ _ _
 **7.** If your username can't be easily mentioned, please set your nickname to something mentionable. For example, "ᐃᑦᑎᓂᖅᓯᐅᑐᖅ ᑕᖅᓴᖅ" isn't particularly useful.
 
 _ _
-**8.** No political / controversial content on this server. If you wish to discuss those head over to: https://discord.gg/q4KdJFP
+**8.** No controversial content, including politics and religion, is allowed.
```

Comment:
```suggestion
**8.** No controversial content (including politics and religion) is allowed.
```

Reply:
No other rules make use of parentheses, and there's no reason to use them over commas.  Pretty sure it actually makes the sentence more informal.

Reply:
What about placing the `is allowed` in front of listing? imo that would make it more natural to read (that's pretty much what in theory parentheses would achieve as well).
Something like
```suggestion
**8.** No controversial content is allowed, including politics and religion.
```
Eventually
```suggestion
**8.** No controversial content is allowed - including politics and religion.
```